src/com/cloudogu/ces/cesbuildlib/Trivy.groovy

```diff
@@ -0,0 +1,133 @@
+package com.cloudogu.ces.cesbuildlib
+
+class Trivy implements Serializable {
+    static final String DEFAULT_TRIVY_VERSION = "0.57.1"
+    private script
+    private Docker docker
+    private String trivyVersion
+    private String trivyDirectory = "trivy"
+
+    Trivy(script, String trivyVersion = "0.57.1", Docker docker = new Docker(script)) {
+        this.script = script
+        this.trivyVersion = trivyVersion
+        this.docker = docker
+    }
+
+    /**
+     * Scans an image for vulnerabilities.
+     * Notes:
+     * - Use a .trivyignore file for allowed CVEs
+     * - This function will generate a JSON formatted report file which can be converted to other formats via saveFormattedTrivyReport()
+     * - Evaluate via exit codes: 0 = no vulnerability; 1 = vulnerabilities found; other = function call failed
```

Comment:
It has to be exit code 10 for vulnerabilities or am I not understanding the code correctly?
```suggestion
     * - Evaluate via exit codes: 0 = no vulnerability; 10 = vulnerabilities found; other = function call failed
```

Reply:
This seems to be an old comment which doesn't apply any more as the function now has a boolean return value

Reply:
Done, see https://github.com/cloudogu/ces-build-lib/commit/cd604e834ad121321c5eda5362a42d62bccdcb15